src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+h1,
+h2,
+ol,
+p {
+  margin: 0;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 17px;
+}
+
+h2 {
+  font-weight: normal;
+  font-size: 15px;
+  padding: 1em 0;
```

Comment:
this is better to set as margins since it's an indentation from the nearby container